engine/triangulation/dim3/surfaces.cpp

```diff
@@ -242,13 +242,20 @@ bool Triangulation<3>::hasSplittingSurface() {
     if (splittingSurface_.known())
         return splittingSurface_.value();
 
-    // In the main loop of this procedure, we will assume the triangulation is connected.
-    // If it isn't, we see if each component has a splitting surface.
+    // Until we know otherwise, we'll assume there's no splitting surface.
+    splittingSurface_ = false;
```

Comment:
It won't affect this code, but splittingSurface_ is a Property<bool>, which means it can be unknown, or a valid bool (either true or false). Until you know whether a splitting surface exists, you should not set this value.

Reply:
That is a good convention to adhere to; I will conform to that.